src/main/java/nextstep/security/authorization/web/DenyAllAuthorizationManager.java

```diff
@@ -0,0 +1,13 @@
+package nextstep.security.authorization.web;
+
+import jakarta.servlet.http.HttpServletRequest;
+import nextstep.security.authentication.Authentication;
+import nextstep.security.authorization.AuthorizationDecision;
+import nextstep.security.authorization.AuthorizationManager;
+
+public class DenyAllAuthorizationManager implements AuthorizationManager<HttpServletRequest> {
+    @Override
+    public AuthorizationDecision check(Authentication authentication, HttpServletRequest object) {
```

Comment:
```suggestion
    public AuthorizationResult check(Authentication authentication, HttpServletRequest object) {
```
반환 타입들도 잘 추상화해주신 클래스로 만들어주면 좋겠네요 :)